### ISSUE TITLE
[SUGGESTION] Auto detect armor for armor underwear

### ISSUE THREAD

**Issue by Sunconure11** [Wabbit0101/mods_hoardercraft#72]
At the very least, a config option for this would be very useful.

**Comment by Sunconure11** [comment]
This is definitely possible, at least. The Betweenlands has an item that auto-detects valid armor and weaponry that it can use for crafting (the red, blue, and green gems).

**Comment by Wabbit0101** [comment]
This will be added as an option that must be enabled. It won't be done automatically as that would break the ability to associated the functionality the advanced tiers or rewards-only armors. Closing as it's already been requested #64. Will be adding to 1.14 version.

**Comment by Sunconure11** [comment]
1.14 probably isn't going to be stable for a while. Forge 1.14 is apparently a nightmare to work with and 1.14 MC appears to be very buggy.

I was merely asking though due to the fact I have a lot of armor in my pack alone.

**Comment by Wabbit0101** [comment]
Yes. I (unfortunately) started mangling the library and mod after the first release because I thought Forge 1.13 would be released soon. I honestly did not have any intention of releasing another 1.12.2 version. Things have not worked out that way. If it's really important I can see to doing a patch to what's there...let me know.

**Comment by Sunconure11** [comment]
I haven't found any bugs so far, but I am currently unable to check, something is throwing a stack overflow error with chests. This has also hindered my capacity to add all armor in my pack to the current lists, since the bug fires mid-game.

**Comment by Sunconure11** [comment]
![Screenshot 2019-07-02 11 06 51](https://user-images.githubusercontent.com/5614902/60523892-8c67b380-9cb9-11e9-8b0d-80ab32b9a6cd.png)

Working on my modpack, by painstakingly adding every set of armor in the pack to the lists, and I found this. It would appear Aether Legacy has issues with this.

**Comment by Wabbit0101** [comment]
Ok. Will check.

**Comment by Sunconure11** [comment]
Did you ever find out what caused this?

Also, for 1.12 at least, I think it would be okay for you to add auto-armor detection, despite TaN being abandoned.

**Comment by Wabbit0101** [comment]
Hmm. I tried this w/ the usual config updates and the regular and auto-adjust temperature liners seem to work (JEI preview and actually work work). As a test I updated the config to include liners for obsidian and valkyrie armor. The version of The Aether I used is: **aether_legacy-1.12.2-v1.4.4**. Anything else I can try?
![72_1of4](https://user-images.githubusercontent.com/15458867/70467626-8f9a7300-1a93-11ea-8d4f-30f07381a8c3.png)
![72_2of4](https://user-images.githubusercontent.com/15458867/70467655-9c1ecb80-1a93-11ea-9ebf-f04f77d800b3.png)
![72_3of4](https://user-images.githubusercontent.com/15458867/70467662-9f19bc00-1a93-11ea-977f-c04d63fe1eef.png)
![72_4of4](https://user-images.githubusercontent.com/15458867/70467668-a17c1600-1a93-11ea-8e34-47c148719e82.png)

The test config files are attached.
[wabbity_armorunder.cfg.txt](https://github.com/Wabbit0101/mods_hoardercraft/files/3941494/wabbity_armorunder.cfg.txt)

**Comment by Sunconure11** [comment]
[wabbity_armorunder.cfg.txt](https://github.com/Wabbit0101/mods_hoardercraft/files/3941895/wabbity_armorunder.cfg.txt)
This is my config file.
